topics/statistics/tutorials/classification_regression/tutorial.md

```diff
@@ -0,0 +1,215 @@
+> 3. {% icon param-select %} *"Select input type"*: `tabular data`
+> 4. {% icon param-file %} *"Training samples dataset"*: `train_data`
+> 5. {% icon param-check %} *"Does the dataset contain header"*: `Yes`
+> 6. {% icon param-select %} *"Choose how to select data by column"*: `All columns but by column header name(s)`
+> 7. {% icon param-text %} *"Type header name(s)"*: `target`
+> 8. {% icon param-file %} *"Dataset containing class labels"*: `train_data`
+> 9. {% icon param-check %} *"Does the dataset contain header"*: `Yes`
+> 10. {% icon param-select %} *"Choose how to select data by column"*: `Select columns by column header name(s)`
+> 11. {% icon param-text %} *"Select target column(s)"*: `target`
+> 12. `Execute` the classifier to train
+{: .hands_on}
+
+
+## Predict using a trained model
+The previous step produces a model file of type `zip`. Rename this file to `model.zip` by using `edit` dataset property. The trained model is used to predict the categories of each row in `test_data` dataset.
+
+> ### {% icon hands_on %} Hands-on: Predict using a trained model
+> 
+> **Gradient Boosting Regressor** {% icon tool %} with the following parameters
+> 
+> 1. {% icon param-select %} *"Select a Classification Task"*: `Load a model and predict`
+> 2. {% icon param-file %} *"Models"*: `model.zip`
+> 3. {% icon param-file %} *"Data (tabular)"*: `test_data`
+> 4. {% icon param-check %} *"Does the dataset contain header"*: `Yes`
+> 5. {% icon param-select %} *"Select the type of prediction"*: `Predict class labels`
+> 6. `Execute` to predict categories
+{: .hands_on}
+
+
+## Visualize the predictions
+The previous step produces a predicted file of type `tabular`. Rename this file to `predicted_data` by using `edit` dataset property. This tool will create three output files, one each for true vs predicted values, scatter plot for true and predicted values and a residual plot. These files are zipped. These files should be downloaded and unzipped to see the `html` file which contains the plot.
+
+> ### {% icon hands_on %} Hands-on: Visualize the predictions
+> 
+> **Plot actual vs predicted curves and residual plots** {% icon tool %} with the following parameters
+> 
+> 1. {% icon param-file %} *"Select input data file"*: `test_target`
+> 2. {% icon param-file %} *"Select predicted data file"*: `predicted_data`
+> 4. `Execute` to create visualizations
+>
+>    ![data](images/true_pred_curves.png "True vs predicted targets curves. These curves should be close to each other for a good regression performance.")
+>    ![data](images/true_vs_pred_scatter.png "Scatter plot for true vs. predicted targets. The data points (blue) should be close to the orange curve (y = x) which shows that the true and predicted values are close.")
+>    ![data](images/residual_plot.png "Residual plot between residual (predicted - true) and predicted targets. For good regression performance, this plot should exhibit a random pattern.")
+{: .hands_on}
+
+
+
```

Comment:
Could you add a conclusion? Like explaining a bit the results, how is it related to the original question, etc
Thanks

Reply:
Added a couple of lines of description after each section